_posts/2019/2019-10-29-Snap-GE-statement.md

```diff
@@ -0,0 +1,27 @@
+---
+date:         2019-10-29
+category:     blog
+tags:
+  - Elections
+layout:       post
+title:        "General Election Statement"
+image:        /assets/img/articles/passport.jpg
+author:       Harley Faggetter
+---
+
+Dear Pirates,
+
+Pirate Party UK believes that leaving the EU without a deal goes against our Party principles. In particular, we believe it will result in the likely degradation of civil liberties, particularly through past Tory campaigns to water down or replace the Human Rights Act, and also digital rights and freedoms given through the EU, such as the protections of the GDPR and access to the Horizon 2020 Open Access programmes.
+
+As a result, in this particular election, we feel that the primary driver for voters is likely to be around our future relationship with the EU. Rather than potentially splitting the vote, as a party that has historically had rather differing views internally on our future relationship with the EU, we should not stand in this election to allow our members to vote freely for the parties and candidates they believe either most represents their views on our future relationship with Europe, or are more likely to win against a party or candidate that would seek the no deal we believe to be incompatible to party values.
```

Comment:
```suggestion
In this particular election, we feel that the primary driver for voters is likely to be around our future relationship with the EU. As a party that has historically had differing views internally on our future relationship with the EU, we feel that we should allow our members to vote freely in this election for the parties and candidates they believe most represent their views on this single issue. Thus, rather than potentially splitting the vote, we have decided that we should not stand any Pirate Party candidates in this General Election, making it more likely that parties or candidates that seek a 'no deal' exit or have approaches that are incompatible with PPUK values will be defeated in this single-issue election.
```